### ISSUE TITLE
Failed bus connection error is not informative

### ISSUE THREAD

**Issue by timbset** [reimagined/resolve#618]
I'm trying to connect to rabbitmq bus using incorrect URL. The following error is thrown:
```
Unhandled rejection Error: connect ECONNREFUSED localhost:5672
```

**Comment by TatyanaRyzh** [comment]
Hi timbset,

We've reviewed your Pull Request #650 and will merge it as soon as possible. Once it's done, we will let you know here.

Thank you very much for helping us improve reSolve!

**Comment by TatyanaRyzh** [comment]
Fixed in [0.14.0](https://github.com/reimagined/resolve/releases/tag/V0.14.0)
